src/shapes.js

```diff
@@ -176,7 +176,7 @@ export const FilterShape = PropTypes.shape({
 export const LocationShape = PropTypes.shape({
   search: PropTypes.string,
   pathname: PropTypes.string.isRequired,
-}).isRequired;
```

Comment:
Er det greit om denne fjernes? Bedre å legge den på der den faktisk er required

Reply:
høres greit ut for meg